### ISSUE TITLE
Incorrect create table definition in ImportSchema.sql (tableName -> `tableName`)

### ISSUE THREAD

**Issue by Sengchheang** [FluidTYPO3/fluidcontent#175]
Current

CREATE TABLE IF NOT EXISTS cf_extbase_datamapfactory_datamap (
    id int(11) unsigned NOT NULL auto_increment,
    identifier varchar(250) DEFAULT '' NOT NULL,
    expires int(11) unsigned DEFAULT '0' NOT NULL,
    content mediumblob,
    PRIMARY KEY (id),
    KEY cache_id (identifier,expires)
) ENGINE=InnoDB;

CREATE TABLE IF NOT EXISTS cf_extbase_datamapfactory_datamap_tags (
    id int(11) unsigned NOT NULL auto_increment,
    identifier varchar(250) DEFAULT '' NOT NULL,
    tag varchar(250) DEFAULT '' NOT NULL,
    PRIMARY KEY (id),
    KEY cache_id (identifier),
    KEY cache_tag (tag)
) ENGINE=InnoDB;

Should be

CREATE TABLE IF NOT EXISTS `cf_extbase_datamapfactory_datamap` (
    id int(11) unsigned NOT NULL auto_increment,
    identifier varchar(250) DEFAULT '' NOT NULL,
    expires int(11) unsigned DEFAULT '0' NOT NULL,
    content mediumblob,
    PRIMARY KEY (id),
    KEY cache_id (identifier,expires)
) ENGINE=InnoDB;

CREATE TABLE IF NOT EXISTS `cf_extbase_datamapfactory_datamap_tags`(
    id int(11) unsigned NOT NULL auto_increment,
    identifier varchar(250) DEFAULT '' NOT NULL,
    tag varchar(250) DEFAULT '' NOT NULL,
    PRIMARY KEY (id),
    KEY cache_id (identifier),
    KEY cache_tag (tag)
) ENGINE=InnoDB;

Long story short, table name should be surround by ``. 
Found this also happens in fluidpages, flux, vhs too.

**Comment by tomahi** [comment]
Hello,

I still have this issue in TYPO3 6.2 installtool:
![grafik](https://cloud.githubusercontent.com/assets/16877853/26547007/2a6bad04-446d-11e7-8d15-5b84da9c47c0.png)

I manually added the bugfix from Flux you posted above, to the file fluidcontent\Classes\Backend\TableConfigurationPostProcessor.php (which was added in Version 4.3.0) and it solved the issue. As we fetch the extension by composer, it is just a temporary solution. Could you please add the fix to a TYPO3 6.2 compatible version?
In fact I couldn't even find the fix in the current version (but I have no instances to test if the issue also appears in TYPO3 7 and 8). So probably this version needs that fix too.

Kind regards,
Steffen

**Comment by NamelessCoder** [comment]
Hi Steffen,

This issue has been solved for almost three years and we no longer release 6.2-compatible versions. In fact, from now on we won't be releasing even 7.6-compatible releases for fluidcontent.

And the fix simply does not apply to 6.2 because it uses constants introduced later than this (FYI it works by coincidence but does not do what it is supposed to do; is always true).

I will close this issue now and strongly recommend you upgrade to a current LTS version with 8.7 being the recommended version that lets you pull the most recent versions of Fluidcontent and other extensions.
